snippets/product-grid-item.liquid

```diff
@@ -37,10 +37,7 @@
   {% assign sold_out = false %}
 {% endif %}
 
-{% comment %}
-  Set a class for sold-out and on-sale items
-{% endcomment %}
-<div class="grid__item {{grid_item_width}}{% if sold_out %} sold-out{% endif %}{% if on_sale %} on-sale{% endif %}">
+<div class="grid__item {{grid_item_width}}">
```

Comment:
Coding style question: do we not include spaces in a Liquid output tag when no filters are applied?